models/identity.go

```diff
@@ -74,8 +74,13 @@ func FindIdentityByIdAndProvider(tx *storage.Connection, providerId, provider st
 
 // FindIdentitiesByUser returns all identities associated to a user
 func FindIdentitiesByUser(tx *storage.Connection, user *User) ([]*Identity, error) {
```

Comment:
nit: why not just rename this function to `FindIdentitiesByUserID` ?

Reply:
Ah it's not being used a lot. Renamed!